Dockerfile

```diff
@@ -27,14 +26,8 @@ RUN curl -sSL https://github.com/resin-io/qemu/commit/db186a3f83454268c43fc793a4
 # - resulting binaries are located in "/usr/local/bin/qemu-*"
 ARG TARGET_ARCH=arm-linux-user,aarch64-linux-user,ppc64le-linux-user,riscv64-linux-user
 RUN mkdir build \
- && cd build \
- && ../configure --static --target-list=$TARGET_ARCH  \
- && make -j $(nproc) install
+    && cd build \
+    && ../configure --static --target-list=$TARGET_ARCH  \
+    && make -j $(nproc) install
 
-FROM busybox
```

Comment:
Why did you remove the much smaller final stage?

Reply:
You right. `busybox` is smaller than `debian`. And if it does matter, I can fixed it in PR (return previous structure).

I was guided by idea of gap between `qemu-register` and `user-implementation`. And if you build your own implementation you can use large or smaller base image (qemu-register) from you copy binary files & `register.sh`.

Please let me know what do think about that.